### PR TITLE
docs: Added docstring for `COO.mT`

### PR DESCRIPTION
**Pull request by DeaMariaLeon**
Could you please tell me if the docstrings should be like this?

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -59,6 +59,7 @@ plugins:
               show_if_no_docstring: true
               members_order: source
               docstring_style: numpy
+              show_source: false
```
Comment (hameerabbasi) [inline mkdocs.yml:62]:
```suggestion
```

I'd keep the source visible

File: docs/operations.md [hunk 2]
```diff
@@ -77,7 +77,7 @@ In these cases, they will produce an output with a fill value of `1` or `True`,
 assuming the original array has a fill value of `0` or `False` respectively.
 
 If densification is needed, it must be explicit. In other words, you must call
-[`sparse.SparseArray.todense`][] on the [`sparse.SparseArray`][] object. If both operands are [sparse.SparseArray][],
+[`sparse.SparseArray.todense`][] on the [`sparse.SparseArray`][] object. If both operands are [`sparse.SparseArray`][],
```
Comment (hameerabbasi) [inline docs/operations.md:80]: Thanks for the fix.

File: sparse/numba_backend/_coo/core.py [hunk 3]
```diff
@@ -851,6 +851,38 @@ def T(self):
...
+        See Also
+        --------
+        - [`sparse.COO.transpose`][] :
+            A method where you can specify the order of the axes.
+        - [`numpy.ndarray.mT`][] :
+            Numpy equivalent property.
+
+        Examples
+        --------
+
```
Comment (hameerabbasi) [inline sparse/numba_backend/_coo/core.py:872]:
```suggestion
```

Unless the newline is needed for proper rendering, I'd remove it.

**Review by hameerabbasi — CHANGES_REQUESTED** [review]
Some minor nits, but everything else LGTM.

One comment: I'd copy the descriptions from the [Array API docs](https://data-apis.org/array-api/latest/) as much as I can.

File: sparse/numba_backend/_coo/core.py (hunk 3 above)
Comment (DeaMariaLeon) [reply sparse/numba_backend/_coo/core.py]: Done.

**Comment by DeaMariaLeon** [comment]
>I'd copy the descriptions from the [Array API docs](https://data-apis.org/array-api/latest/) as much as I can.

I changed the description then. 

I think I addressed the rest of the comments too. Please let me know if something else is needed.

**Comment by hameerabbasi** [comment]
Thanks for the work on this @DeaMariaLeon! This should merge once CI passes.

**Pull request merged**